x-pack/filebeat/input/s3/config.go

```diff
@@ -0,0 +1,23 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package s3
+
+import "github.com/elastic/beats/filebeat/harvester"
+
+var defaultConfig = config{
+	ForwarderConfig: harvester.ForwarderConfig{
+		Type: "s3",
+	},
+}
+
+type config struct {
+	harvester.ForwarderConfig `config:",inline"`
+	QueueURLs                 []string `config:"queueURLs" validate:"nonzero,required"`
+	BucketNames               []string `config:"bucketNames"`
```

Comment:
I guess this parameter is minded for security reasons? Would it be ok to just trust anything incoming from SQS?

Reply:
Yeah I was just thinking maybe users have different s3 buckets setup to send message to one SQS for different reasons. Then they might want to specify only collecting logs from specific buckets instead of all...

Reply:
You could argue this may be a bad pattern? that means that a Filebeat input will be fetching messages from SQS and ignoring them, hiding them from other input that has real interest on those buckets.

I would expect different queues for different concerns?

Reply:
Yeah I don't have much experience there but I do agree with you.